### ISSUE TITLE
Run-time notice with Cache_Lite_File::get()

### ISSUE THREAD

**Issue by GoogleCodeExporter** [chrsmithdemos/minify#32]
```
I have the latest version of Minify (2.0), and am getting this run-time notice 
in my logs:

"Declaration of Cache_Lite_File::get() should be compatible with that of 
Cache_Lite::get() in 
Minify.php on line 291"

It's not a critical issue to fix, but fixing it would at least ensure the best 
interoperability and forward 
compatibility of the PHP 5 code.  Thanks!
```

---

Original issue reported on code.google.com by `stein.j...@gmail.com` on 29 May 2008 at 7:25

**Comment by GoogleCodeExporter** [comment]
```
Yep. Cache_Lite_File isn't E_STRICT. I'm considering boiling it down to a single
class for best performance. Or doing the same with Zend_Cache. With all the real
world use Cache_Lite has received I'm hesitant to jump on Zend_Cache just 
because
it's "best practice" code.
```

---

Original comment by `mrclay....@gmail.com` on 30 May 2008 at 1:36
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
I am getting this error as well. And in my case, this error outputs in my 
m.php/css
output, so it becomes basically gibberish because it generates all these  
"Warning:
Cannot modify header information - headers already sent" .

Any way to remove this error without messing with the error output in PHP?
```

---

Original comment by `FT.MikeL...@gmail.com` on 31 May 2008 at 7:02

**Comment by GoogleCodeExporter** [comment]
```
[deleted comment]
```

**Comment by GoogleCodeExporter** [comment]
```
On line 912 of /lib/Cache/Lite/File.php is the method get():
http://code.google.com/p/minify/source/browse/tags/release_2.0.0/lib/Cache/Lite/
File.php#912

In the parameter list, add a third parameter, $thisValueIgnored = false, so the 
line
reads:

    function get($id, $group = 'default', $thisValueIgnored = false)

This change appeases the E_STRICT gods and will be included in 2.0.1.


```

---

Original comment by `mrclay....@gmail.com` on 1 Jun 2008 at 2:43
- Changed state: **Fixed**
